.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
   - "2.7"
+  - "3.4"
```

Comment:
> I don't think we should support 3.2 because very few people use it and 3.2 compatibility is more complicated: the `u'unicode string'` was removed in 3.0 and only [readded](https://www.python.org/dev/peps/pep-0414/) in 3.3.

Ok, that sounds reasonable. Why don't you update the README with a list of supported Python versions so that it's clear.

> Given that travis relies on volunteers for computing resources

I wouldn't worry about that too much. Millions of projects use Travis, and us using a few more builds is not going to make a difference.

> I think 2.7, 3.3, and nightly should be enough, and maybe 2.6 if that is necessary.

Given my above reasoning, how about `2.6`, `3.3`, `3.4`, `3.5`, `3.5-dev`, and `nightly`? I think `2.6` might not have `codecov`. If that's the case, omit the `codecov` step. I'm not too keen on `2.6`, so if it's causing lots of problems, don't worry about it. But the other versions I listed I think we should try to support if possible.